bin/git-forgit

```diff
@@ -539,22 +540,27 @@ _forgit_checkout_commit() {
         FZF_DEFAULT_OPTS="$opts" fzf | eval "$_forgit_extract_sha" | xargs -I% $git_checkout % --
 }
 
+_forgit_branch_delete_preview() {
+    git log "$1" "${_forgit_log_preview_options2[@]}"
+}
+
 _forgit_branch_delete() {
     _forgit_inside_work_tree || return 1
-    local git_branch preview opts cmd branches
+    local git_branch opts branches
     git_branch="git branch $FORGIT_BRANCH_DELETE_GIT_OPTS"
     [[ $# -ne 0 ]] && { $git_branch -D "$@"; return $?; }
-    preview="git log {1} $_forgit_log_preview_options"
 
     opts="
         $FORGIT_FZF_DEFAULT_OPTS
         +s --multi --tiebreak=index --header-lines=1
-        --preview=\"$preview\"
         $FORGIT_BRANCH_DELETE_FZF_OPTS
     "
 
-    cmd="git branch --color=always | LC_ALL=C sort -k1.1,1.1 -rs"
-    branches=$(eval "$cmd" | FZF_DEFAULT_OPTS="$opts" fzf | awk '{print $1}')
+    branches=$(
+        git branch --color=always |
+        LC_ALL=C sort -k1.1,1.1 -rs |
+        FZF_DEFAULT_OPTS="$opts" fzf --preview="$FORGIT branch_delete_preview {1}" |
```

Comment:
Hey @sandr01d, is there a reason why you moved the `--preview` option from the `opts` variable to the `fzf` argument list? This way we cannot override the preview in `FORGIT_BRANCH_DELETE_FZF_OPTS` anymore.

Reply:
No, that appears to be a mistake on my side and we should move it back. Good catch! I don't have time to work on this today or tomorrow, so feel free to push a commit to this PR. Otherwise I'll change it back the next time I get around to it (probably Saturday).

Reply:
I've moved the `--preview` option back into the `opts` variable where applicable. I think we should also do so for the functions where it never was in the `opts` variable (`_forgit_blame`, `_forgit_revert_commit` and `_forgit_stash_push`), but I would like to keep this PR focused on removing deferred execution and will do so in a follow up PR.